<launch>
  <node pkg="ros_pcduino" name="avoidance" type="irobot_avoidance" output="screen" >
    <param name="paused" type="bool" value="true" />
    <param name="maxLinear" type="double" value="0.3" />
    <param name="maxAngular" type="double" value="1.5" />
    <param name="goalRotationDepth" type="int" value="450" />
    <param name="goalFrontDepth" type="int" value="450" />
    <param name="minRotationPoints" type="int" value="50" />
    <param name="maxRotationDepth" type="int" value="800" />
    <param name="minFrontPoints" type="int" value="300" />
    <param name="maxFrontDepth" type="int" value="1000" />
    <param name="frontWidth" type="int" value="300" />
    <param name="frontHeight" type="int" value="40" />
    <param name="rotationWidth" type="int" value="100" />
    <param name="rotationHeight" type="int" value="60" />
    <param name="isSonarOn" type="bool" value="false" />
    <param name="minSonarFront" type="double" value="0.40" />
  </node>
</launch>
